lib/manager/cocoapods/artifacts.ts

```diff
@@ -106,7 +106,7 @@ export async function updateArtifacts({
 
   const podsDir = join(dirname(packageFileName), 'Pods');
   const podsManifestFileName = join(podsDir, 'Manifest.lock');
-  if (await platform.getFile(podsManifestFileName)) {
+  if (await readLocalFile(podsManifestFileName, 'utf8')) {
```

Comment:
Does this always return or throw?

Reply:
It returns `null` if not found

Reply:
So i would suggest the following, cause we don't need extra `string` parsing
```suggestion
  if (await readLocalFile(podsManifestFileName)) {
```